libraries/resource_helpers.rb

```diff
@@ -42,6 +42,16 @@ def platform_service_name
       platform_family?('rhel', 'amazon', 'fedora') ? "postgresql-#{node.run_state['postgresql']['version']}" : 'postgresql'
     end
 
+    def postgresql_service
+      resources(service: 'postgresql')
```

Comment:
Can't we just use `platform_service_name`? rather than failing out?

Reply:
I don't understand what you want me to do / what you are saying ?
I have used this trick (inspired by andy on slack) because when you declare a service resource in custom resource and you notify it in other resource, the other resource don't find the service.
May be there is better to do that, and I want to know !